tests/chainer_tests/test_variable.py

```diff
@@ -1854,6 +1932,43 @@ def test_copy_to_initialize(self):
         x.initialize((3, 2))
         assert x.data is y.data
 
+    def test_set_array_numpy(self):
```

Comment:
Please separate those tests and use `inject_backend_tests` just like `TestVariableSetArray`.